deeprank/learn/NeuralNet.py

```diff
@@ -71,11 +69,7 @@ def __init__(self, data_set, model,
                 Only applicable on 'class' task.
```

Comment:
What would class weights be?

Reply:
I guess this should depend on the frequency of variants that occur of each class. Low for benign, because it's common? High for disease, because it's rare?

See also: https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html

Reply:
So in the dataset from BioProdict, we have more benign variants evidently:
Benign- 54,138
Pathogenic- 43,460
Although 40% of the data has pathogenic labels, class weights can be higher for it and lower for benign.

Reply:
We can sample the data to create a subset that has 50% benign and 50% pathogenic.